### DISCUSSION TITLE
Get 400 Bad Request when connector.activate of Arb Sepolia

### DISCUSSION THREAD

**Discussion by ducndt** [WalletConnect/walletconnect-monorepo#4126] [Developer Support] [answered]
Hi every one, i see the arbitrum sepolia (testnet) is add via this issue
https://github.com/WalletConnect/walletconnect-monorepo/issues/3454

But when i use the useSwitchNetwork of this chain, it throws the error 
> We don't support the chainId you provided: eip155:421614

so i make the issue to ask everyone to check the error. Does it my mistake when setup chains or the mistake of intect Arbitrum sepolia chain? 
Thanks in advance. 

my chain.ts
```
421614: {
    urls: [getInfuraUrlFor("arbitrum-sepolia"), "https://sepolia-rollup.arbitrum.io/rpc"].filter(
      Boolean,
    ),
    name: "Arbitrum Sepolia Rollup Testnet",
    nativeCurrency: ETH,
    blockExplorerUrls: ["https://sepolia.arbiscan.io"],
  },
```

**Comment by glitch-txs** [comment, marked as answer]
you'll need to use a different RPC provider since ours doesn't support Arbitrum Sepolia

**Reply by ducndt** [reply]
Thanks @glitch-txs .So now i need to make a issue and add Arbitrum Sepolia into your chainlist?

**Reply by glitch-txs** [reply]
This is actually not related to WalletConnect v2 protocol, we do support this chain,this is related to our Blockchain API, but you can replace it with a public RPC or something like Infura or Alchemy
https://docs.walletconnect.com/cloud/blockchain-api#supported-chains

Could you share a MRE? is this MetaMask only?

**Reply by ducndt** [reply]
Can you explain more about MRE for me?

**Reply by glitch-txs** [reply]
minimal reproducible example
something like this should actually work, could you test here + share the steps to reproduce the issue?
https://stackblitz.com/edit/stackblitz-starters-g9ruyh?file=app%2Flayout.tsx
